Makefile

```diff
@@ -292,11 +295,17 @@ clean-downloadcache:
 update-downloadcache: $(CACHE)
 	cd $(CACHE) && git pull origin master || true
 
+ifeq ($(LSB_RELEASE),precise)
```

Comment:
It is done to create an alternative set of dependencies for the GUI for precise, since it does not support wheels.  The process would be driven by running 'make package' in the juju-gut-charm directory while on a precise machine.  The resulting charm with proper dependencies would be uploaded to the charm store only for the precise series.